src/clndr.js

```diff
@@ -205,10 +211,11 @@
                 if (this.options.lengthOfTime.startDate) {
                     this.intervalStart =
                         moment(this.options.lengthOfTime.startDate)
-                            .startOf('day');
+                            .startOf('week');
```

Comment:
I changed to this to `week` to enforce any errors arising from users trying to set a weekly calendar to the middle of the week. That causes breakage in the calendar so this should correct any of those errors. cc @kylestetz